jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/OdePropertiesTest.java

```diff
@@ -80,13 +70,8 @@ public void testSettersAndGetters() {
             int testTrustRetries = 23;
             String testKafkaTopicOdeBsmPojo = "testKafkaTopicOdeBsmPojo";
             String testKafkaTopicOdeBsmJson = "testKafkaTopicOdeBsmJson";
-            String testVersion = "1.1.0-SNAPSHOT";
             int testImportProcessorBufferSize = 83;
 
-            String[] testKafkaTopicsDisabled = new String[] { "testKafkaTopicsDisabled0" };
-            Set<String> testKafkaTopicsDisabledSet = new HashSet<>();
-            testKafkaTopicsDisabledSet.add("testKafkaTopicsDisabledSet0");
```

Comment:
note: now all moved to [OdeKafkaProprtiesTest](https://github.com/CDOT-CV/jpo-ode/pull/110/files#diff-b771b678401cde76ebe39b58c11c2ce155b9092e84fbfba935a4555cb609728b)